packages/dashboard/src/managers/rmf-health-state-manager.ts

```diff
@@ -0,0 +1,74 @@
+import { DoorSummary, LiftSummary, DispenserSummary, RobotSummary } from 'react-components';
+
+export interface HealthStatus {
+  door: DoorSummary;
+  lift: LiftSummary;
+  dispenser: DispenserSummary;
+  robot: RobotSummary;
+}
+
+export default class RmfHealthStateManager {
+  getDoorSummary = (): DoorSummary => {
+    return {
+      item: 'Door',
+      itemSummary: { operational: 2, outOfOrder: 1 },
+      spoiltDoors: [
+        {
+          itemNameAndState: 'hardware_door - unknown',
+          door: {
+            name: 'hardware_door',
```

Comment:
Does it cause any bugs or inconsistency if a non-existing door is named here? If so, better to just provide an empty array.

Reply:
Ignore this if you move the new main menu into a new temp file and revert the main-menu changes.